eos-tech-support/eos-convert-system-qa

```diff
@@ -154,8 +154,18 @@ if $OSTREE || $APPS; then
             --set=branches="${new_branch};"
 
         # Check for an update every time the updater runs
-        # rather than once every two weeks
-	sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater.conf
+        # rather than once every two weeks. eos-updater.conf is the old
+        # location of the config file.
+        if [ -f /etc/eos-updater.conf ]; then
```

Comment:
So, hmm, this code is not correct in the case that `/etc/eos-updater/eos-autoupdater.conf` and `/etc/eos-updater.conf` both exist, since `eos-autoupdater` loads the former (new location) in preference to the latter (old location). So running `sed` on the old location won’t help.

What we want to do is:
 1. If the new location exists, patch that.
 2. Otherwise, if the old location exists, patch that.
 3. Otherwise, copy the defaults to the new location and patch that.